src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceFiducialLocatorConfigurationWizard.java

```diff
@@ -0,0 +1,233 @@
+package org.openpnp.machine.reference.vision.wizards;
```

Comment:
I'd like to see the wizard simplified a bit. I think we can remove the "Use custom pipeline?" checkbox and just use the custom one if it's been created, otherwise use the default. If the user wants to go back to using the default they can just reset it. Resetting it should just set the custom one to null.